### DISCUSSION TITLE
https://theme-hope.vuejs.press/zh/cookbook/markdown/demo.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#906] [Announcements]
# https://theme-hope.vuejs.press/zh/cookbook/markdown/demo.html

提示 上方的代码中 这里 后面有两个空格 提示 上方的代码中换行后面有也两个空格 提示 更多详见

https://theme-hope.vuejs.press/zh/cookbook/markdown/demo.html

**Comment by 732124645** [comment]
图片是怎么出现标注的呀

**Reply by chairowell** [reply]
```md
![这里填写你对图片的标注](这里是你的图片地址)
```
